user-service/src/main/java/com/ecommercepractice/userservice/controller/UserController.java

```diff
@@ -0,0 +1,55 @@
+package com.ecommercepractice.userservice.controller;
+
+import com.ecommercepractice.userservice.models.User;
+import com.ecommercepractice.userservice.service.UserService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequestMapping("api/v1/user")
+@RestController
+@Slf4j // Lombok
+public class UserController {
+    private final UserService userService;
```

Comment:
Please annotate userService with @Autowired and remove this
 @Autowired
    public UserController(UserService userService) {
        this.userService = userService;
    }

Reply:
Ok